packages/server-core-integration/src/lib/configManifest.ts

```diff
@@ -76,12 +102,16 @@ export interface SubDeviceConfigManifestEntry extends ConfigManifestEntryBase {
 export interface TableConfigManifestEntry extends ConfigManifestEntryBase {
 	/** Whether this follows the deviceId logic for updating */
 	isSubDevices?: boolean
+	/** The default name/id for any new devices */
 	subDeviceDefaultName?: string
+	/** The type any new entry gets by default */
 	defaultType?: string
 	type: ConfigManifestEntryType.TABLE
+	/** Used when the .config indexes are different from the type enum */
 	deviceTypesMapping?: any
+	/** The name of the the property used to decide the type of the entry */
 	typeField?: string
-	/** Only one type means that the option will not be present */
+	/** Only one type means that the type option will not be present */
```

Comment:
Would be good to include the same comment about the `disabled` property as in `meteor/lib/api/deviceConfig.ts` I think.

Reply:
Definitely!